melange.opam.template

```diff
@@ -1,3 +1,10 @@
 pin-depends: [
   [ "melange-compiler-libs.dev" "git+https://github.com/melange-re/melange-compiler-libs.git"]
 ]
+
+install: [
+  [ "dune" "install" "-p" "%{name}%" ]
+  [ "./fix-install-opam.sh" "%{_:lib}%" "%{_:build}%" ]
+]
+
+setenv: [ BSLIB = "%{_:lib}%/melange" ]
```

Comment:
@anmonteiro ha, opam has a `setenv`. But I don't think this is needed anymore @roddyyaga there was some changes on how the stdlib can be located. Not a problem to have it here, tho

Reply:
You mean BSLIB doesn't need to be set?

Reply:
Yup, related PR https://github.com/melange-re/melange/pull/131

Reply:
Cool, if you mean it's not needed because of the default setting of `<bsc-path>/../lib/ocaml` I think opam does need to set it, since the right path there is `<bsc-path>/../lib/melange/melange` (opam has `.../{lib,bin}/package_name` where esy has `.../package_name/{lib,bin}`).